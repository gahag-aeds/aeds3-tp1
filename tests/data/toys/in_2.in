3
0.0000 0.98625 0.20166 
0.01375 0.0000 0.08397 
0.79834 0.91603 0.0000 
