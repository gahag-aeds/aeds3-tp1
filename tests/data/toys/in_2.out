0.221772 0.024040 0.754188 